docs/storybook/src/components/MessageCenterField.stories.tsx

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 import type { Decorator, Meta, StoryObj } from "@storybook/react";
 import { AppUiDecorator, InitializerDecorator } from "../Decorators";
-import { MessageCenterField } from "@itwin/appui-react/src/appui-react/statusfields/MessageCenter";
+import { MessageCenterField } from "@itwin/appui-react/src/appui-react/statusfields/message-center/MessageCenterField";
```

Comment:
The visuals look great, however it would be great if we could see different states of the message center field (i.e. stories for empty, different types of messages, etc): https://itwin.github.io/appui/746/?path=/story/components-messagecenterfield--basic
Additionally it would be great to add a story which would contain some UI to manually add/clear the messages (similarly to what we have in test-app Component Examples), i.e. adding a sb decorator to one of the stories https://storybook.js.org/docs/writing-stories/decorators